README.md

```diff
@@ -1,66 +1,108 @@
-1. Install docker toolbox from https://www.docker.com/products/docker-toolbox
-2. Run the docker quickstart terminal
-3. Change default vm settings (Only needed once)
-  1. Remove the default vm
+## Working with Network Proxies
+If you are working behind a corporate firewall, you must provide a sockets based proxy solution in the host OS. [Proxycap](http://www.proxycap.com)  is recommended.
```

Comment:
I'm also not sure if we want to officially recommend proxycap. Brian what do you think?